regnety/dataset/imagenet.py

```diff
@@ -48,24 +47,25 @@ def __init__(
             raise ValueError("List of TFrecords paths cannot be None or empty")
         self.tfrecs_filepath = tfrecs_filepath
         self.batch_size = batch_size
-        self.image_size = image_size
+        self.image_size = tf.constant([image_size] * self.batch_size) 
         self.augment_fn = augment_fn
         self.num_classes = num_classes
         self.randaugment = randaugment
         if self.randaugment:
-            self._augmenter = RandAugment(magnitude=5, num_layers=2)
+            self.strength = 5
```

Comment:
What does `strength` indicate in our setting?

Reply:
Similar to PAWS-TF, here strength influences the delta for color jitter ops.

Reply:
Let's have a comment on that. Also, this is a motivation from SimCLR actually. The strength of augmentation transforms was fairly studied in that paper.